source/extensions/filters/http/jwt_authn/authenticator.cc

```diff
@@ -123,6 +126,17 @@ std::string AuthenticatorImpl::name() const {
   return "_UNKNOWN_";
 }
 
+std::string AuthenticatorImpl::convertDoubleOrIntToString(const double& double_value) {
+  double fraction;
+  if ((double_value < 0 ||
+       double_value->number_value() >= static_cast<double>(std::numeric_limits<uint64_t>::max())) &&
+      modf(double_value, &fraction) != 0) {
```

Comment:
this is integ_part, not fraction part

Reply:
`modf(double_value, &fraction)` this will check the fraction part.

Reply:
From the document of `modf(double,  &integer_part)`,  the 2nd argument is returning the integer part of the double.    The variable name of `fraction` is misleading, it should be `integer_part`